postgres/server/init.sls

```diff
@@ -164,13 +164,12 @@ postgresql-pg_hba:
     # Create the empty file before managing to overcome the limitation of check_cmd
     - onlyif: test -f {{ pg_hba_path }} || touch {{ pg_hba_path }}
```

Comment:
This is no longer needed since we are going to remove `check_cmd` parameter. Same applies for the `pg_ident`.